packages/web3-shared/src/pipes/index.ts

```diff
@@ -40,6 +41,11 @@ export function resolveNetworkChainId(networkType: NetworkType) {
     }
 }
 
+export function resolveChainNetwork(chainId: ChainId) {
```

Comment:
Remove it?

Reply:
Need it, now getChainName or resolveChainName returns `'Ethereum Mainnet'` this kind of long name, which is not compatible of redpacket payload schema.